pkg/operator/ceph/object/rgw.go

```diff
@@ -102,13 +103,24 @@ func (c *clusterConfig) startRGWPods(realmName, zoneGroupName, zoneName string)
 		c.store.Spec.Gateway.Instances = 1
 	}
 
+	rgwsToSkipReconcile, err := controller.GetDaemonsToSkipReconcile(c.clusterInfo.Context, c.context, c.clusterInfo.Namespace, config.RgwType, AppName)
+	if err != nil {
+		return errors.Wrap(err, "failed to check for RGWs to skip reconcile")
+	}
+
 	// start a new deployment and scale up
 	// We force a single deployment and later set the deployment replica to the "instances" value
 	desiredRgwInstances := 1
 	for i := 0; i < desiredRgwInstances; i++ {
 		var err error
 
 		daemonLetterID := k8sutil.IndexToName(i)
+
+		if rgwsToSkipReconcile.Has(daemonLetterID) {
+			logger.Warningf("skipping reconcile of rgw daemon %q with label %q", daemonLetterID, cephv1.SkipReconcileLabelKey)
+			return nil
```

Comment:
This seems to allow partially re-conciliated rgws daemons in case we have many ... is this behavior correct?

Reply:
yes, because we only want to skip the reconcile for specific daemons